### PR TITLE
Update project structure and remove old files

### PR DESCRIPTION
**Pull request by AustinShalit**
The Jenkins build tools need to be updated for this to pass: https://github.com/wpilibsuite/build-tools/pull/30

### CODE REVIEW

**Comment by PeterJohnson** [comment]
Is there any reason not to remove the WPIJavaCV files?  I can tell they're definitely old due to the com.googlecode imports..

**Comment by bradamiller** [comment]
I agree with removing them.

**Comment by 333fred** [comment]
Case sensitive file systems strike again: FakeRobot/Readme.txt is different from fakeRobot/code.

**Comment by 333fred** [comment]
For that matter, any reason that fakeRobot can't just be another sourceset in the main project?

File: build.gradle [hunk 1]
```diff
@@ -33,6 +33,17 @@ dependencies {
     compile 'jfree:jfreechart:1.0.13'
 }
 
+project(':fakeRobot') {
```
Comment (333fred) [inline build.gradle]: Please update the WPILibVersion plugin while you do this. It should look like this: https://github.com/wpilibsuite/allwpilib/blob/master/build.gradle#L3-L12

**Comment by AustinShalit** [comment]
Updated the WPILibVersion Plugin

**Comment by 333fred** [comment]
@frcjenkins retest this please

**Comment by AustinShalit** [comment]
@frcjenkins retest this please

**Comment by 333fred** [comment]
@frcjenkins retest this please

**Pull request merged**